code/modules/mob/living/simple_animal/hostile/mining_mobs/ice demon.dm

```diff
@@ -85,3 +85,15 @@
 	if(prob(5))
 		new /obj/item/raw_anomaly_core/bluespace(loc)
 	return ..()
+
+
+//SKYRAT CHANGE, NERFED ICE DEMON FOR MINING
```

Comment:
This could probably be added in a modular file, but also like does it matter idfk

Reply:
modularity causes me agony

Reply:
Modularization and code standards are for wussies.